src/components/Chart.js

```diff
@@ -0,0 +1,15 @@
+import Highcharts from 'highcharts' // Highcharts is being used under the Creative Commons Attribution-NonCommercial 3.0 license
+import highchartsAccessibility from 'highcharts/modules/accessibility'
+import highchartsExporting from 'highcharts/modules/exporting'
+import HighchartsReact from 'highcharts-react-official'
+import { h } from 'react-hyperscript-helpers'
+
+
+highchartsAccessibility(Highcharts)
+highchartsExporting(Highcharts)
+
+const Chart = ({ options }) => {
+  return h(HighchartsReact, { highcharts: Highcharts, options })
+}
```

Comment:
Do we want to one-line this to make it a little more obvious that it's a straight wrapper?
```suggestion
const Chart = ({ options }) => h(HighchartsReact, { highcharts: Highcharts, options })
```